### DISCUSSION TITLE
Should I suggest a UI like armoury crate? - I mean it looks good although its buggy

### DISCUSSION THREAD

**Discussion by giannispomonis** [seerge/g-helper#1489] [Ideas]
If only you could just build a UI like armoury's crate UI. It would be amazing to have a functional clean app like g-helper also fancy to always permanently show in our Zephyrus Duo laptops on the secondary screen. I mean dont change anything of its functions, just a fancier UI with wild graphics demonstrating circuits and fans. Just my idea... please dont judge me. I already use  g-helper but I just miss the appearance of armoury crate filling my secondary screen in my DUO 16

**Comment by Scarlaid** [comment]
I guess the first thing is make it appealing for custom shell theme first
![image](https://github.com/seerge/g-helper/assets/90851437/89d18f97-2b07-46ce-a30d-56415986ab0c)
